merlin/legacy/inference/triton/ensemble.py

```diff
@@ -0,0 +1,783 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import json
+import os
+import warnings
+from shutil import copyfile, copytree
+
+import numpy as np
+import tritonclient.grpc.model_config_pb2 as model_config
+from google.protobuf import text_format
+
+from merlin.core.dispatch import is_string_dtype
+from merlin.schema import Tags
+from nvtabular import ColumnSelector
+
+
+def export_tensorflow_ensemble(
```

Comment:
similar to the data conversions module, looks like we already have these in [merlin.systems.triton.export](https://github.com/NVIDIA-Merlin/systems/blob/v0.8.0/merlin/systems/triton/export.py#L34)

Reply:
Yeah, a lot of this is there, but trying to figure out what is and isn't the same, which parts of the Systems code have changed/evolved, etc has been stopping us from removing the old code from NVT. Putting the old code from NVT verbatim in a completely separate standalone package bypasses that concern. If we care enough to do the work of figuring it out and consolidating we can do that, but my plan is to let this code sit here for a few months while we sort out session-based serving and then delete the whole package.